scripts/romanisim-make-stack

```diff
@@ -92,8 +85,8 @@ def main():
     # SCA detector identifier number
     parser.add_argument('-s', '--sca',
                         type=int,
-                        default=0,
-                        help='SCA to simulate')
+                        default=-1,
```

Comment:
Did you ensure that this behaved as expected for both a specified SCA and for all?

Reply:
I only tested the "commands" mode, but this works for both single and multiple SCAs at present.